graph-migration/README.md

```diff
@@ -5,30 +5,41 @@
 Graph Schema Migration tool creates legacy graph frame. Based on legacy graph frame schema, the tool generates native graph schema.
 
 There are two types of schema, cql and gremlin formats. Cql format schema is the CQL script to create graph keyspace and
-tables. Gremlin format is native graph schema.
-Run the following command to generate native graph schema
+tables. Gremlin format is groovy script to create native graph schema under Gremlin console.
+
+Run the following command to generate native graph schema with DSE server is up:
+```
+dse graph-migrate-schema [-cql | -gremlin] <legacy_graph> <native_graph>
 ```
-dse graph-migrate-schema -cql|-gremlin <legacy_graph> <native_graph>
+
+- `-cql`: specifies the output as cql format.
+- `-gremlin`: specifies the output as gremlin format.
+
+Example:
+```
+dse graph-migrate-schema -gremlin reviewerRating nreviewerRating > native_schema.groovy
 ```
 
-All properties are single. Multi properties are migrated as a single property. Meta properties, Index and MVs are are
+Please note that the following are NOT supported:
+- Meta properties
+- Multi properties
+- Indexes
+
+All properties are single. Multi properties are migrated as a single property. Meta properties, Indexes, and MVs are
 dropped. If custom handling of meta properties and multi properties, renaming properties and dropping properties are
-needed, user should modify the generated native graph schema. The native graph schema can be created through cqlsh or
-gremlin.
+needed, users should modify the generated native graph schema.
 
-Then pass generated schema to cqlsh or gremlin-console depends on the type
+To create the native graph schema, pass the generated schema to cqlsh or gremlin-console depends on the type
 
 Example:
 ```
-dse graph-migrate-schema -gremlin reviewerRating nreviewerRating > native_schema.groovy
 dse gremlin-console -e native_schema.groovy
 ```
 
 ## Migrate Data
 
-Provided script is just an example of the data migration process.
-It should work as is, if new native schema was generated with previous step and was applied without 
-modifications. 
+The provided script is just an example of the data migration process.
+It should work with the native graph schema generated in the previous step, assuming there is no modification to it.
```

Comment:
I assume the generated script would also work after being modified, provided the modification are correct? I'm actually not sure here, more of a question than a statement :)

Reply:
I made the changes based on the original paragraph. I guess what we try to say here is that the data migration part should work if there is no modification made to the script after it is generated. Otherwise, the data migration would not work because of the modification.

Reply:
I prefer to state here, something like:
if you modify generated schema, please modify the migration script accordingly. 
And there are some example later, like getLegacyPropertyName() could be overridden for name changes.

Reply:
Sure, took in your suggestion. 😄